documentdb/spring-data-azure-documentdb/src/main/java/com/microsoft/azure/spring/data/documentdb/repository/support/DocumentDbRepositoryFactoryBean.java

```diff
@@ -23,18 +22,12 @@
         implements ApplicationContextAware {
 
     private ApplicationContext applicationContext;
-    private DocumentDbOperations operations;
-
 
     public DocumentDbRepositoryFactoryBean(Class<? extends T> repositoryInterface) {
 
         super(repositoryInterface);
     }
 
-    public void setDocumentDbOperations(DocumentDbOperations operations) {
```

Comment:
pls don't remove this method. spring data will need the setOperation to set operations